build.sh

```diff
@@ -377,6 +377,10 @@ fi
 if hasArg --incl-cache-stats; then
     BUILD_REPORT_INCL_CACHE_STATS=ON
 fi
+if hasArg pylibraft; then
+    COMPILE_LIBRARY=ON
+    CMAKE_TARGET="${CMAKE_TARGET};raft_lib"
```

Comment:
Is it ok to append multiple times the same target? If one now specifies `pylibraft --compile-lib` then `raft_lib` will exist twice. Perhaps a safer thing to do is have the `hasArg pylibraft` in https://github.com/rapidsai/raft/blob/03d97fd50cd35969c32bf24b9025a8012ed8399e/build.sh#L308, e.g.:

```
if hasArg --compile-lib || hasArg pylibraft || (( ${NUMARGS} == 0 )); then
```

Reply:
True, this proposed solution also avoid code duplication. Thanks!